robottelo/constants/__init__.py

```diff
@@ -1865,8 +1865,6 @@
         'setup': entities.ConfigGroup,
         'skip_for_ui': True,
     },
```

Comment:
When removing the puppet-related bookmarks, you might want to remove the ConfigGroup too. It is also puppet-related and fails on default 6.11 setup.